server/components/Release.tsx

```diff
@@ -151,6 +151,12 @@ export function Release({ release, releaseMap }: { release: HarmonyRelease; rele
 						<td>{formatScriptFrequency(script)}</td>
 					</tr>
 				)}
+				{types && types.length > 0 && (
+					<tr>
+						<th>Types</th>
+						<td>{types.join(' + ')}</td>
```

Comment:
This is what I meant by displaying the "original source values" aka the alternative property values for each provider:
```suggestion
						<td>
							{types.join(' + ')}
							<AlternativeValues
								property={(release) => release.types}
								display={(types) => types.join(' + ') || '[none]'}
							/>
						</td>
```

Reply:
This is great, works as it should

![grafik](https://github.com/kellnerd/harmony/assets/29852/e542bfc7-3623-47e2-a6e4-682828aa0aa3)